extensions/arc/src/models/postgresModel.ts

```diff
@@ -73,9 +73,9 @@ export class PostgresModel extends ResourceModel {
 		const ramLimit = this._config.spec.scheduling?.default?.resources?.limits?.memory;
 		const cpuRequest = this._config.spec.scheduling?.default?.resources?.requests?.cpu;
 		const ramRequest = this._config.spec.scheduling?.default?.resources?.requests?.memory;
-		const dataStorage = this._config.spec.storage?.data?.size;
-		const logStorage = this._config.spec.storage?.logs?.size;
-		const backupsStorage = this._config.spec.storage?.backups?.size;
+		const dataStorage = this._config.spec.storage?.data?.volumes?.[0]?.size;
```

Comment:
@nasc17 @bergeron This is the only thing I'm unsure of - what should we be doing if there are multiple volumes?

Reply:
Postgres will only have 1 volume.  I'm not sure why it was expanded into an array.   It could be that sqlmi supports multiple volumes, but I don't know for sure.